src/components/settings/NotificationsSettings/index.tsx

```diff
@@ -70,70 +70,72 @@ const NotificationsSettings: React.FC = () => {
       >
         <SettingsHeader title="Notifications" />
         <MobileHeader title="Notifications" back="/settings" />
-        <div className="NotificationsSettings__wrapper">
-          <SettingsItem
-            title="Display App on Discover Page"
-            subtitle="Provide the domain of your app"
-            className="NotificationsSettings__notifications"
-          >
-            <Input
-              value={filterAppDomain}
-              placeholder="app.example.com"
-              onChange={ev => updateSettings({ filterAppDomain: ev.target.value })}
-            />
-          </SettingsItem>
+        <div className="NotificationsSettings__content">
```

Comment:
Just added new container div. Would be better to have checked with `Hide whitespace` feature while reviewing in GH